third_party/BUILD

```diff
@@ -92,3 +93,10 @@ java_binary(
     visibility = ["//visibility:public"],
     runtime_deps = ["//third_party/jvm/org/pantsbuild:jarjar"],
 )
+
+java_import(
```

Comment:
Note - I really want to separate the android and jvm stuff so a pure jvm project doesn't get android shit dragged into it. Totally for the future, but just a thing to keep in mind.

Reply:
We should be pulling the jar from the android sdk, not the jar from tools. It ignores the toolchain and sdk, so this is suspicious. It's on the list.

Reply:
Nod.